components-core/src/test/java/com/adyen/checkout/components/core/internal/util/NumberExtensionTest.kt

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2023 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by ararat on 21/11/2023.
+ */
+
+package com.adyen.checkout.components.core.internal.util
+
+import org.junit.Assert.assertEquals
+import org.junit.Test
+import java.util.Locale
+
+internal class NumberExtensionTest {
+
+    @Test
+    fun `integer format formats with en-US locale`() {
+        val locale = Locale.forLanguageTag("en-US")
+
+        assertEquals("1", 1.format(locale))
```

Comment:
This is unnecessary in this case specifically but check out `@ParameterizedTest` if you didn't know about it. It might be nice for these types of tests and we have a few examples of it in the codebase.

Reply:
Let's keep it like this then 👍